svc-lighthouse-api/src/test/resources/application-test.properties

```diff
@@ -1,9 +1,34 @@
-abd-data-access.rabbitmq.claim-submit-exchange=claim-submit-exchange
-abd-data-access.rabbitmq.claim-submit-queue=claim-submit
-abd-data-access.rabbitmq.claim-submit-routing-key=code.#
+# application.properties is loaded, then this file is loaded
 
-lh.tokenurl=${LH_TOKEN_URL}
-lh.fhirurl=${LH_FHIR_URL}
-lh.pemkey=${LH_PRIVATE_KEY}
-lh.assertionurl=${LH_ASSERTION_URL}
-lh.clientId=${LH_ACCESS_CLIENT_ID}
+lh.tokenurl=https://sandbox-api.va.gov/oauth2/health/system/v1/token
+lh.fhirurl=https://sandbox-api.va.gov/services/fhir/v0/r4
+lh.assertionurl=https://deptva-eval.okta.com/oauth2/aus8nm1q0f7VQ0a482p7/v1/token
+
+lh.clientId=00ah28rb5jLj4NeSN2p7
```

Comment:
`clientId` was moved from [LighthouseApiServiceTest.java](https://github.com/department-of-veterans-affairs/abd-vro/pull/817/files#diff-bdbbe42db095dbfa263768864064afe3d3ff18a269b4c196c6f6875680b6545a):
https://github.com/department-of-veterans-affairs/abd-vro/pull/817/files#diff-bdbbe42db095dbfa263768864064afe3d3ff18a269b4c196c6f6875680b6545aL41

Reply:
Those I put in the test file should not be any problem.